components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -1012,17 +1013,14 @@ def load_execution_container_name(self):
             # Accept configured value for debug purposes
             return
 
-        os_release = dotenv_values(self._os_release_file_path)
-        if "ubuntu" == os_release["ID"]:
-            self.execution_container = (
-                f"clp-execution-x86-{os_release['ID']}-{os_release['VERSION_CODENAME']}:main"
-            )
-        else:
-            raise NotImplementedError(
-                f"Unsupported OS {os_release['ID']} in {OS_RELEASE_FILE_PATH}"
-            )
+        if self._image_id_path.exists():
+            with open(self._image_id_path) as image_id_file:
+                self.execution_container = image_id_file.read().strip()
```

Comment:
we may change this attribute name to `package_container` though i feel the name `execution_container` is more intuitive to those who are unfamiliar with the history of the execution container